conan/tools/files/files.py

```diff
@@ -263,6 +264,47 @@ def chdir(conanfile, newdir):
         os.chdir(old_path)
 
 
+def chmod(conanfile, path:str, read:bool, write:bool, execute:bool, recursive:bool=False):
```

Comment:
```suggestion
def chmod(conanfile, path:str, read:bool=None, write:bool=None, execute:bool=None, recursive:bool=False):
```
Using `None` to mean "unchanged" for a `chmod +x` or `chmod +w` equivalent would provide a very useful and intuitive interface, e.g. `chmod(self, some_exe, execute=True)`?